pkg/webhooks/trainingruntime_webhook.go

```diff
@@ -54,14 +55,22 @@ func (w *TrainingRuntimeWebhook) ValidateCreate(ctx context.Context, obj apirunt
 }
 
 func validateReplicatedJobs(rJobs []jobsetv1alpha2.ReplicatedJob) field.ErrorList {
+	validReplicatedJobNameSet := map[string]struct{}{
+		constants.JobInitializer: {},
+		constants.JobLauncher:    {},
+		constants.JobTrainerNode: {},
+	}
+
```

Comment:
Actually, the requirement was recently changed in https://github.com/kubeflow/trainer/pull/2523
We need to check if rJob has dedicated ancestor label instead of rJob name.

Reply:
Thanks for your suggestion, I'll take a deeper look at the reference you provided and revise it tomorrow.

Reply:
I have updated the UT and check the replica with ancestor label, PTAL.

cc @tenzen-y